src/app/modules/asset-level/asset-level-edit/asset-level-edit.component.ts

```diff
@@ -27,14 +27,16 @@ export class AssetLevelEditComponent implements OnInit {
   ];
 
   constructor (
+    @Inject(MAT_DIALOG_DATA) private data: { assetLevelId: number },
+    private readonly dialogRef: MatDialogRef<AssetLevelEditComponent>,
     private readonly formBuilder: FormBuilder,
     private readonly assetLevelService: AssetLevelService,
-    private readonly modalEditAssetLevel: MdbModalRef<AssetLevelEditComponent>,
     private readonly notificationService: NotificationService,
     private readonly authService: AuthService
   ) {}
 
   ngOnInit(): void {
+    this.assetLevelId = this.data.assetLevelId;
```

Comment:
esta variable es medio al pedo, podrias usar this.data.assetLevelId en todos lados y listo porque ya es una variable global